src/app/models/treeoperations.h

```diff
@@ -33,6 +33,8 @@ class TreeOperations : public QObject, public ConnectionsTree::Operations
 
     void notifyDbWasUnloaded(int dbIndex) override;
 
+    void openDeleteKeyDialog(ConnectionsTree::KeyItem& key) override;
```

Comment:
See my comment https://github.com/uglide/RedisDesktopManager/pull/3632/commits/60077de8c38ed33cf6f89f0d6ee1f1a28a15ae0b#r68562294

This method should perform actual key removal instead of redirecting it to ValueEditor